### PR TITLE
How about a money green?

### PR DESCRIPTION
**Pull request by hampelm**
I had to take out the `section` to get the green underlines to align right 

Also I've been getting really into utility CSS classes over component styles, trying that out here. 

Screen Shot 2021-02-28 at 11 18 00 AM

### CODE REVIEW

**Comment by hampelm** [comment]
It also looks great with just plain white 

Screen Shot 2021-02-28 at 11 21 20 AM

**Comment by jessicamcinchak** [comment]
Cool, nice to see how you're using utility CSS classes here and glad to follow that going foward!

I like green background for money sections, but then maybe bold black links instead of current blue? Or a navy blue? Otherwise I'd say white background with our current hue for links just for contrast sake.

Otherwise looks good so approving now, happy to defer to your final color decision here!

**Review by jessicamcinchak — APPROVED** [review]

**Pull request merged**
